SpeechTranslator/AudioCapturePermissions.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.Media.Capture;
+
+namespace SpeechTranslator
+{
+    class AudioCapturePermissions
+    {  // If no recording device is attached, attempting to get access to audio capture devices will throw 
+        // a System.Exception object, with this HResult set.
+        private static int NoCaptureDevicesHResult = -1072845856;
+
+        /// <summary>
+        /// On desktop/tablet systems, users are prompted to give permission to use capture devices on a 
+        /// per-app basis. Along with declaring the microphone DeviceCapability in the package manifest,
+        /// this method tests the privacy setting for microphone access for this application.
+        /// Note that this only checks the Settings->Privacy->Microphone setting, it does not handle
+        /// the Cortana/Dictation privacy check, however (Under Settings->Privacy->Speech, Inking and Typing).
+        /// 
+        /// Developers should ideally perform a check like this every time their app gains focus, in order to 
+        /// check if the user has changed the setting while the app was suspended or not in focus.
+        /// </summary>
+        /// <returns>true if the microphone can be accessed without any permissions problems.</returns>
+        /// 
+        public async static Task<bool> RequestMicrophonePermission()
```

Comment:
change this method name to RequestMicrophoneCapture ... there is no permission on Iot Core